ui-tests/src/test/java/org/openmrs/reference/DeleteRequestAppointmentTest.java

```diff
@@ -34,9 +34,8 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore //See RA-1216 for details
     @Category(BuildTests.class)
-    public void deleteRequestAppointmentTest() throws Exception {
```

Comment:
this isn't a fix, it's ignoring the test dear @HerbertYiga

Reply:
> this isn't a fix, it's ignoring the test dear @HerbertYiga

one quick question, do we have any thing like Travis that checks through the tests here?